graphite/graphite.go

```diff
@@ -56,7 +56,9 @@ func (r *Request) Query(host string, header http.Header) (Response, error) {
 		Path:     "/render/",
 		RawQuery: v.Encode(),
 	}
-	if u, _ := url.Parse(host); u.Scheme != "" && u.Host != "" {
+
+	u, err := url.Parse(host)
+	if err== nil && u.Scheme != "" && u.Host != "" {
```

Comment:
Worth adding the missing space here?
```suggestion
	if err== nil && u.Scheme != "" && u.Host != "" {
```

Maybe we should also add the style check to the GitHub action. Not sure why it's not complaining. It should. 🤔

Reply:
Fixed the style issues